.changeset/brown-items-reflect.md

```diff
@@ -0,0 +1,6 @@
+---
+'@clerk/clerk-js': minor
+'@clerk/types': minor
+---
+
+accountless
```

Comment:
Once we're ready, let's expand the changeset description here. And consolidate the two changeset files.